sql/core/src/test/resources/sql-tests/inputs/udf/pgSQL/udf-select_having.sql

```diff
@@ -6,7 +6,7 @@
 -- https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/sql/select_having.sql
 --
 -- This test file was converted from inputs/pgSQL/select_having.sql
--- TODO: We should add UDFs in GROUP BY clause when [SPARK-28445] is resolved.
+-- TODO: We should add UDFs in GROUP BY clause when [SPARK-28386] and [SPARK-26741] is resolved.
```

Comment:
SPARK-28445 was wrong from the beginning, @shivusondur ?

Reply:
@dongjoon-hyun 
After resolving the SPARK-28445 also, test were failing and found [SPARK-28386] and [SPARK-26741] are blocking it.

for furher details follow https://github.com/apache/spark/pull/25215#issuecomment-518897609